backtrace-library/src/main/java/backtraceio/library/models/json/BacktraceAttributes.java

```diff
@@ -199,4 +195,21 @@ private void convertAttributes(Map<String, Object> attributes) {
             }
         }
     }
+
+    public String getApplicationName() {
+        return this.context.getApplicationInfo().loadLabel(this.context
+                .getPackageManager()).toString();
+    }
+
+    public String getApplicationVersion() throws PackageManager.NameNotFoundException {
```

Comment:
This code was duplicated (maybe you already moved it here?). Was a Konrad comment too